### ISSUE TITLE
Getting error code EsE4KPE02 when clicking + on joined chats and the screen is on direct messages

### ISSUE THREAD

**Issue by cleydyr** [debiki/talkyard#31]
(Not sure if those are the steps to reproduce on a clean instance, but this is roughly what I did on a test site)
# Steps to reproduce
1. Create two users A and B
2. Login with two of them on different browsers
3. Send a message from A to B
4. As B, go to "Direct Messages" screen using the left-hand sidebar
5. As B, click on the + icon near "Joined sites".

# Expected behavior
A direct message will show on the bottom of the window.

# Current behavior 
Browser shows a modal with the following message:
```
EsE4KPE02

See Dev Tools for details: usually Ctrl + Shift + C, here in the browser, then click Console.
```

**Comment by kajmagnus** [comment]
Thanks for reporting, I can reproduce this. The problem is that, when looking at a direct message, one isn't currently located in any specific site-section ("site-section"? Usually there's just 1 site-section, but one can also create different site-sections, which works a bit like Reddit sub communities).

However, when clicking `(+)`, the app attempts to create the new chat channel in the current site-section — which is nothing, so this check fails: 

```
var category = debiki2.store_getCurrOrDefaultCat(store);   <— oops, finds nothing
debiki2.dieIf(!category, 'EsE4KPE02');
```

Ok so I'll need to use the usually-just-exactly-one site-section & category, and this'll then work fine.

**Comment by kajmagnus** [comment]
Ok, fixed now, ... in one of my my work-in-progress GitHub branch (`w-km3` today).  I'll code review & incl in the next version. (I'll leave this open until merged & deployed)

**Comment by kajmagnus** [comment]
@cleydyr Now I've released a new version with this fixed. Would you like to see if this works for you now?

Your server ought to have auto upgraded to a version with this fixed (version `v0.6.7-WIP-6-0cc6df74e`).

**Comment by cleydyr** [comment]
I didn't have the opportunity to manually test it. You can close it for now.

Thank you!
